examples/quickstarts/helidon-standalone-quickstart-mp/pom.xml

```diff
@@ -22,11 +22,11 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>io.helidon.examples.quickstarts</groupId>
     <artifactId>helidon-standalone-quickstart-mp</artifactId>
-    <version>2.0.0-SNAPSHOT</version>
+    <version>2.0.0-M3</version>
```

Comment:
The change of version is not correct (for Helidon itself) - we handle that in our release process, to reference the currently released version.

Reply:
Sure. I cannot build the project with "-SNAPSHOT"

Reply:
Unresolving, this must reference the SNAPSHOT version. You need to build the project in order to use SNAPSHOT versions.